tests/test_modeling_common.py

```diff
@@ -589,6 +589,11 @@ def _create_and_check_torchscript(self, config, inputs_dict):
                     traced_model = torch.jit.trace(
                         model, (input_ids, attention_mask, decoder_input_ids, decoder_attention_mask)
                     )
+                elif 'bbox' in inputs and 'image' in inputs: # LayoutLMv2 requires additional inputs
```

Comment:
I'm not exactly happy with how this is done. However, it seemed like a big change to add another field to the config